pkg/handlers/adminapi/office_users.go

```diff
@@ -326,6 +326,26 @@ func (h CreateOfficeUserHandler) Handle(params officeuserop.CreateOfficeUserPara
 				return officeuserop.NewCreateOfficeUserUnprocessableEntity(), err
 			}
 
+			privilegesAllowed, verrs, err := h.UserPrivilegeAssociator.VerifyUserPrivilegeAllowed(appCtx, payload.Roles, payload.Privileges)
+
+			if err != nil {
+				appCtx.Logger().Error("Error verifying user privileges allowed", zap.Error(err))
+				appCtx.Logger().Error(err.Error())
+				return officeuserop.NewCreateOfficeUserUnprocessableEntity(), err
+			}
```

Comment:
`verrs` is missing its err check if `verrs.HasAny()`. Like:
```suggestion
			privilegesAllowed, verrs, err := h.UserPrivilegeAssociator.VerifyUserPrivilegeAllowed(appCtx, payload.Roles, payload.Privileges)

			if err != nil {
				appCtx.Logger().Error("Error verifying user privileges allowed", zap.Error(err))
				appCtx.Logger().Error(err.Error())
				return officeuserop.NewCreateOfficeUserUnprocessableEntity(), err
			}
			if verrs.HasAny() {
				validationError := &adminmessages.ValidationError{
					InvalidFields: handlers.NewValidationErrorsResponse(verrs).Errors,
				}

				validationError.Title = handlers.FmtString(handlers.ValidationErrMessage)
				validationError.Detail = handlers.FmtString("The information you provided is invalid.")
				validationError.Instance = handlers.FmtUUID(h.GetTraceIDFromRequest(params.HTTPRequest))

				return officeuserop.NewCreateOfficeUserUnprocessableEntity().WithPayload(validationError), verrs
			}
```